jsonschema2pojo-cli/src/main/scripts/simplelogger.properties

```diff
@@ -0,0 +1,34 @@
+# SLF4J's SimpleLogger configuration file
```

Comment:
This file should be coming from the classpath I think, not read as a file. It should be in the root of the classpath for this cli project e.g. `src/main/resources`.

Reply:
If possible the CLI logging should be configured via command line switches.

Reply:
I will leave the command line switches for a future effort but agree.